docs/modules/ROOT/pages/guides/v4-migration/index.adoc

```diff
@@ -0,0 +1,294 @@
+[[v4-migration]]
+
+= 4.0.0 Migration
+
+NOTE: Version 4.0.0 of the library has not yet been released. However, we recommend making these changes early in order to avoid issues in the future.
+
+This document lists all breaking changes from version 3.x.y to 4.0.0 and how to update.
+
+== How to upgrade
+Simply update `@neo4j/graphql` using npm or your package manager of choice:
+
+[source, bash, indent=0]
+----
+npm update @neo4j/graphql
+----
```

Comment:
Wouldn't it be super nice if we could have that tab-like interface which shows how to do the same thing with different package managers?

Reply:
Yeah that would be cool. I'm not very familiar with adoc so I don't know how/if that's possible. I haven't been able to find anything obvious having just had a quick look at their docs.

Also, this was just copied over from previous migration guides but part of me thinks maybe we shouldn't include this section for now as it's not yet possible to upgrade and I don't want to confuse people. What do you think?